projects/media-viewer/src/lib/annotations/comment-set/comment/comment.component.ts

```diff
@@ -35,6 +35,10 @@ export class CommentComponent implements OnChanges {
   @ViewChild('form') form: ElementRef;
   @ViewChild('textArea') textArea: ElementRef;
 
+  constructor() {
+    this.MAX_COMMENT_LENGTH = 48;
```

Comment:
is this best practise, just want to know

Reply:
the other way is not supported in typescript <3.1.0, which JUI use